packages/node_modules/@node-red/editor-client/src/js/user.js

```diff
@@ -74,7 +74,7 @@ RED.user = (function() {
                     }
                     $('<div class="form-row" style="text-align: right; margin-top: 10px;"><span id="node-dialog-login-failed" style="line-height: 2em;float:left;" class="hide">'+RED._("user.loginFailed")+'</span><img src="red/images/spin.svg" style="height: 30px; margin-right: 10px; " class="login-spinner hide"/>'+
                         (opts.cancelable?'<a href="#" id="node-dialog-login-cancel" style="margin-right: 20px;" tabIndex="'+(i+1)+'">'+RED._("common.label.cancel")+'</a>':'')+
```

Comment:
do we also need it on the cancel button (if it's enabled ?)

Reply:
oh yes, you're right. wasn't aware of that button.